payment-circle/src/main/java/org/stellar/anchor/paymentservice/circle/CirclePaymentService.java

```diff
@@ -27,27 +27,33 @@
 import reactor.util.annotation.NonNull;
 import reactor.util.annotation.Nullable;
 
-@Data
 public class CirclePaymentService implements PaymentService {
   private static final Gson gson = new Gson();
+
+  @Getter()
```

Comment:
Lombok is mostly used for object field access to avoid tediously boilerplate code. As a service, it is actually less confusing to use `getNetwork`, `getUrl`, and 'getSecretKey`.

Reply:
Oh ok, I can change this.